### ISSUE TITLE
README.md indicates wrong type for interface parameter on method SetPosition

### ISSUE THREAD

**Issue by danielrodriguezac** [popcornmix/omxplayer#684]
As the title indicates, [https://github.com/popcornmix/omxplayer#setposition](https://github.com/popcornmix/omxplayer#setposition) indicates that parameter 1 is a string (signature 's') but the player actually expects that parameter to be of object_path (signature 'o') [Reference](https://www.alteeve.com/w/List_of_DBus_data_types).

When a string type is sent the player ignores the command, maybe i am missing something?. I just spent the last couple of days scouring my code for bugs trying to figure out why it was not working and it was just a signature mismatch problem.

This is basically a request for clearer documentation to help other people implementing omxplayer control interfaces.

**Comment by popcornmix** [comment]
I assume SetAlpha has the same issue? Can you see any other problems?

**Comment by danielrodriguezac** [comment]
Yes,when using SetAlpha the player expects the first argument to be an object path (whole signature "ox") so the problem is present there as well. No other problems found so far, will certainly notify if i find anything else :)
